dell/src/test/java/org/apache/iceberg/dell/mock/ecs/EcsS3MockRule.java

```diff
@@ -178,4 +163,16 @@ public String bucket() {
   public String randomObjectName() {
     return "test-" + ID.getAndIncrement() + "-" + UUID.randomUUID();
   }
+
+  @Override
+  public void afterEach(ExtensionContext extensionContext) {
+    System.out.println("Bucket removed");
+    cleanUp();
+  }
+
+  @Override
+  public void beforeEach(ExtensionContext extensionContext) {
+    System.out.println("Bucket removed");
```

Comment:
It seems here we are trying to initialise the bucket.

Reply:
@nk1506 if there are no feedbacks, are we good to merge this?